L1Trigger/TrackFindingTracklet/interface/TrackletEventProcessor.h

```diff
@@ -3,14 +3,16 @@
 #define L1Trigger_TrackFindingTracklet_interface_TrackletEventProcessor_h
 
 #include "L1Trigger/TrackFindingTracklet/interface/Timer.h"
-#include "L1Trigger/TrackFindingTracklet/interface/ChannelAssignment.h"
+#include "DataFormats/L1TrackTrigger/interface/TTTypes.h"
```

Comment:
You changed code here from ChannelAssignment to TTTypes. I have a pull request (that you wanted a wait an deploy) that removed the use of ChannelAssignment in this class. I'm not sure why TTTypes are needed and I see that you made ChannelAssignement forward declared. This might make it hard to merge with the changes I made to remove the use of ChannelAssignement inside the pattern recognition code.

Reply:
ChannelAssignment.h includes TTTypes.h, so the latter was being included into TrackletEventProcessor.h indirectly anyway. It is needed as it defines  type tt:Streams, which used used for the data member streamsTrack_.  
In general, it's a good thing to avoid unnecessary #include, so since the #include of ChannelAssignment.h could be replaced by a forward declaration of this class and a #include of the smaller file TTTypes.h,  this seemed a positive change.

Reply:
OK in general I agree - but I'm just concerned that my pull request to remove the dependence all together on ChannelAssigment might be harder to merge. I moved all the use of the stub and track streams out of the track finding code.

Reply:
I'll give you a hand with your PR. I realize we may have introduced some conflicts, so owe you that ...